waterfox/browser/components/WaterfoxGlue.jsm

```diff
@@ -262,32 +262,40 @@ const WaterfoxGlue = {
 
   async _migrateUI() {
     let currentUIVersion = Services.prefs.getIntPref(
-      "browser.migration.version"
+      "browser.migration.version",
+      128
     );
 
     async function enableTheme(id) {
-      AddonManager.getAddonByID(id).then(addon => addon.enable());
+      const addon = await AddonManager.getAddonByID(id);
+      // If we found it, enable it.
+      addon?.enable;
```

Comment:
This smells fishy...is it really some getter with side-effect? In AddonManager.jsm, it seems to be called as a function that returns a promise.

Reply:
Yeah, the previous version called it as a function...though, why it was 'async', I don't know since it doesn't return anything. Perhaps I don't quite 'get' async/await yet :/

Reply:
Should be called as a function, clipped off the brackets by mistake.